packages/components/src/color-palette/index.js

```diff
@@ -66,7 +66,7 @@ export default function ColorPalette( {
 				disableAlpha
 			/>
 		),
-		[ value ]
+		[ value, onChange ]
```

Comment:
Can we just remove the useCallback call renderCustomColorPicker function. It seems to me that this optimization is not really necessary.

Reply:
Good point! 👍

Reply:
@youknowriad I've removed the useCallback and rebased the PR.